Editor/Core/Config/ImportConfiguration.cs

```diff
@@ -221,10 +265,23 @@ public void ImportGame()
                 Debug.LogError($"Error during Import: {e}");
                 return;
             }
-            if (ConfigurationIndex > ConfigurationExecutors.Length)
+            if (ConfigurationIndex > ConfigurationExecutors.Length - 1)
```

Comment:
minor nitpick,  I'd rather use >= here than -1
They are functionality identical here, this is merely preference